README.md

```diff
@@ -80,7 +80,15 @@ import factory from 'trampss-redux-factory'
 export default factory()('id')('api.raw')('todos')
 ```
 
-Data will be stored into `state.api.raw.todos`
+Data will be stored into `state.api.raw.todos`.
+
+### Types
+Types are :
+  - `map` : your state is a hashmap, usefull to bind your API to Redux
```

Comment:
I think it will be clearer to have directly the reducer model here : 
### Types

Types are :
- `map` : your state is a hashmap, usefull to bind your API to Redux with the following reducer model :

```
{
  data: { <key1>: <instance1>, <key2>: <instance2> },
  array: [<instance1>, <instance2>],
  keys: [<key1>, <key2>],
  initialized: true,
}
```
- `uniq` : your state is an object, much simplier, with the following reducer model :

```
{
  data: <instance>,
  initialized: true,
}
```

Default type is `map`.

To see more informations about types, [go to the specific page](./TYPES.md).